geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ServerHandshakeProcessor.java

```diff
@@ -86,8 +74,10 @@ public static void refuse(OutputStream out, String message, byte exception) thro
 
     // Write the server's member
     DistributedMember member = InternalDistributedSystem.getAnyInstance().getDistributedMember();
-
-    writeServerMember(member, dos);
+    HeapDataOutputStream memberDos = new HeapDataOutputStream(Version.CURRENT);
+    DataSerializer.writeObject(member, memberDos);
```

Comment:
Why did you remove the backward-compatibility serialization here?  If we make a change to the serialized form of this object won't this break backward-compatibility?

Reply:
There shouldn't be any change in behavior from this refactor.  I just inlined the writeServerMember and cleaned up the code a bit as the output stream here can never be a VersionedDataStream (you can see where we construct the stream on line 66).  That being said, if there's a bug with the previous code not using the proper serialization here, let me know and we'll get that fixed.